packages/contracts/test/LibExperience.t.sol

```diff
@@ -24,7 +25,50 @@ contract LibExperienceTest is MudV2Test {
 
     uint32[PStat_length] memory exp = LibExperience.getExp(targetEntity);
     for (uint256 i; i < exp.length; i++) {
-      assertEq(exp[i], 1);
+        assertEq(exp[i], 1);
     }
   }
+
+  function testHasExp() public {
+    assertFalse(LibExperience.hasExp(targetEntity));
+  }
+
+  function testSetExp() public {
+    uint32[PStat_length] memory exp = [uint32(10), 20, 30];
+    Experience.set(targetEntity, exp);
+
+    uint32[PStat_length] memory retrievedExp = LibExperience.getExp(targetEntity);
+    for (uint256 i; i < retrievedExp.length; i++) {
+        assertEq(retrievedExp[i], exp[i]);
+    }
+  }
```

Comment:
naming nit: this should be `testGetExp`
you're not really testing `Experience.set`, since tables're tested by MUD internally. `getExp` is a custom library function however